superset/assets/src/explore/controls.jsx

```diff
@@ -1562,24 +1562,6 @@ export const controls = {
     description: t('Compute the contribution to the total'),
   },
 
-  num_period_compare: {
```

Comment:
If deprecating `num_period_compare` and `period_ratio_type`, do we need a db migration for existing charts that have been saved using that feature?